### DISCUSSION TITLE
Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by travlytle** [WLAN-Pi/feedback#63] [General Inquiries]
I've ordered a couple raspberry pi 5's and some of these m.2 hat's for wifi 7.
https://www.aliexpress.us/item/3256806224989235.html?spm=a2g0o.order_list.order_list_main.5.21ef1802dgZOie&gatewayAdapt=glo2usa

My next question is can I use the current image for Pi4 for Pi5? If not when will we see support for Pi 5?

**Comment by jiribrejcha** [comment]
The current image doesn’t support Raspberry Pi 5. It might boot, but you will experience very slow performance and other issues. 

We are testing Raspberry Pi 5, bit it will require Raspberry Pi OS based on Debian Bookworm. It isn’t a priority for us at the moment, but we are exploring it and should eventually support it. No fixed release date yet.
